uat/custom-components/client-python-paho/README.md

```diff
@@ -128,3 +128,6 @@ Receiving properties for the PUBACK is not implemented - Python Paho MQTT publis
 
 2. Subscribing
 Python Paho MQTT does not provide the API to set the Subscription ID in SUBSCRIBE request.
+
+3. Unsubscription
+In Python Paho on_unsubscribe() callback does not provide reason code for MQTTv3, instead zero code will be returned on success.
```

Comment:
Looks like should be 'reason codes' not 'reason code'

In MQTT v3.1.1 UNSUBACK has no reason codes at all 

See http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html#_Toc398718077 


In MQTT v5.0 UNSUBACK provides LIST of reason codes.
See https://docs.oasis-open.org/mqtt/mqtt/v5.0/os/mqtt-v5.0-os.html#_Toc3901187

Reply:
for 1 filter will be 1 code

Reply:
changed in dd3dd38